### ISSUE TITLE
ELIFECYCLE ERR

### ISSUE THREAD

**Issue by atchouangwa** [techiecoder2079/Cake-Website#1]
I am trying to run the code however i reviece an message saying there is errors in the javascript code?

**Comment by techiecoder2079** [comment]
Did you convert all txt files to java files??

**Comment by techiecoder2079** [comment]
Sorry js files

**Comment by atchouangwa** [comment]
Yes I did, are you familiar with google fire base? It’s giving me troubles

**Comment by atchouangwa** [comment]
I get eslint errors and whenever I bypass them, the JS isn’t deployed

**Comment by techiecoder2079** [comment]
Not much as i have just heard about it.. it is used for backend purpose.
